### PR TITLE
Add path module

### PR DESCRIPTION
**Pull request by amc1234**
Tested in lab. This does not include the changes we discussed today, but merging to make things simpler. Will create a new branch to add the changes.

### CODE REVIEW

File: path.py [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+        self.file = open(str(self.fileName), "r")
+
+    def openAppend(self):
+        self.file = open(str(self.fileName), "a")
+
+class PathManagement:
+    def __init__(self, bleObject, navigationObject):
+        self.BLE = bleObject
+        self.navigate = navigationObject
+        self.aruco_id = 0
```
Comment (rs2thoma) [inline path.py]: aruco_id isnt needed

Comment (rs2thoma) [reply path.py]: also the var shouldnt be snake_case

**Review by rs2thoma — APPROVED** [review]

File: path.py [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+        self.navigate.stop()
+        
+        endTime = datetime.now()
+        timeString = endTime - startTime
+        return timeString.total_seconds()
+
+    def getDirection(self, direction):
+        if direction == f'{Commands.FORWARD.value}'.encode():
+            self.navigate.forward()
+            return "forward"
```
Comment (tiger-hao) [inline path.py:123]: it'd be better to have a `Direction` enum for these constants

**Review by tiger-hao — APPROVED** [review]

File: path.py (hunk 2 above)
Comment (amc1234) [reply path.py:123]: I'll add it to my to do list and merge it with the changes that I'll make to the path file after navigation testing.

**Pull request merged**